src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__wrapper {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+  }
+
+  &__image-block {
+    padding: 32px 19px 36px;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__content {
+    height: 200px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: 0 16px 16px;
+  }
+
+  &__name {
+    @include primaryText;
+    font-weight: 500;
+    margin-top: 0;
+    margin-bottom: 4px;
```

Comment:
reset margins for all elements you need at once, and leave here just: `margin-bottom: 4px;`